client/src/components/Collections/PairedListCollectionCreator.vue

```diff
@@ -262,9 +262,13 @@
                                                 aria-haspopup="true"
                                                 aria-expanded="false"></button>
                                             <div class="dropdown-menu">
-                                                <a class="dropdown-item" @click="changeFilters('illumina')">_1</a>
-                                                <a class="dropdown-item" @click="changeFilters('Rs')">_R1</a>
-                                                <a class="dropdown-item" @click="changeFilters('dot12s')">.1.fastq</a>
+                                                <a
+                                                    class="dropdown-item"
+                                                    v-for="(value, key) in commonFilters"
```

Comment:
I struggled with this bit here, but eventually found that the key, value pairs in commonFilters were flipped, so that the key (labeled value here) is the actual forward/reverse strings, and the value (labeled key here) was the name of the object.  

Can anybody help me understand why I had to index into it like this?  TIA!

Reply:
It's just how vue works, not specific to `commonFilters`. It returns (value, key) when looping objects, and (value, index) when looping arrays. see here: https://v2.vuejs.org/v2/guide/list.html

Also trips me up regularly, as the order [`key`, `value`] is so prevalent.